corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py

```diff
@@ -166,13 +159,14 @@ def get_case_ids_for_owner(self, owner_id):
 
     def _get_case_ids_for_owners_without_extensions(self, owner_id):
         if self.is_clean(owner_id):
+            domain = self.restore_state.domain
             if self.restore_state.is_initial:
                 # for a clean owner's initial sync the base set is just the open ids
-                return set(get_open_case_ids(self.restore_state.domain, owner_id))
+                return set(CaseAccessors(domain).get_open_case_ids(owner_id))
             else:
                 # for a clean owner's steady state sync, the base set is anything modified since last sync
-                return set(get_case_ids_modified_with_owner_since(
-                    self.restore_state.domain, owner_id, self.restore_state.last_sync_log.date
+                return set(CaseAccessors(domain).get_case_ids_modified_with_owner_since(
```

Comment:
Yeah, I'm thinking I'll store it as an instance variable of `CleanOwnerCaseSyncOperation`.